lib/copy_peste/command/init_bdd.rb

```diff
@@ -13,9 +13,9 @@ def run()
         config_path = File.join(Require::Path.root, './', 'copy_peste.yml')
         config_path = File.expand_path config_path
         config = YAML::load_file(config_path)
-        if File.directory? config['port_tree_path']
-          @graph_com.display(10, "Loading directory #{config['port_tree_path']}")
-          scan = ScanSystem.new(config['port_tree_path'])
+        if File.directory? config['ports_tree_path']
```

Comment:
Don't you think it's a good idea to put the string 'ports_tree_path' in a variable somewhere ? It appears twice here and one time in copy_peste.rb